integrations/woopra/lib/index.js

```diff
@@ -22,14 +22,11 @@ var Woopra = (module.exports = integration('Woopra')
   .option('cookieName', 'wooTracker')
   .option('cookieDomain', null)
   .option('cookiePath', '/')
-  .option('ping', true)
-  .option('pingInterval', 12000)
   .option('idleTimeout', 300000)
-  .option('downloadTracking', true)
-  .option('outgoingTracking', true)
+  .option('downloadTracking', false)
+  .option('outgoingTracking', false)
+  .option('clickTracking', false)
   .option('outgoingIgnoreSubdomain', true)
-  .option('downloadPause', 200)
-  .option('outgoingPause', 400)
```

Comment:
I see both of these in the GitHub readme configuration table - have they actually been removed from the library? It looks like the integration's default for `downloadPause` matches woopra's, but `outgoingPause` is double what woopra uses by default, so unless they've been removed and woopra no longer reads these settings we should consider leaving these in place.

Reply:
Those have been removed. We're now using beacons to track downloads and outgoing links. The pause is no longer needed.